tests/test_messages.py

```diff
@@ -135,29 +137,30 @@ def test_image_url_invalid():
         ImageUrl('foobar.potato').format
 
 
-@pytest.mark.parametrize(
-    'document_url,media_type,format',
-    [
-        pytest.param(DocumentUrl('foobar.pdf'), 'application/pdf', 'pdf', id='pdf'),
-        pytest.param(DocumentUrl('foobar.txt'), 'text/plain', 'txt', id='txt'),
-        pytest.param(DocumentUrl('foobar.csv'), 'text/csv', 'csv', id='csv'),
-        pytest.param(
-            DocumentUrl('foobar.docx'),
-            'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
-            'docx',
-            id='docx',
-        ),
-        pytest.param(
-            DocumentUrl('foobar.xlsx'),
-            'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
-            'xlsx',
-            id='xlsx',
-        ),
-        pytest.param(DocumentUrl('foobar.html'), 'text/html', 'html', id='html'),
-        pytest.param(DocumentUrl('foobar.md'), 'text/markdown', 'md', id='md'),
-        pytest.param(DocumentUrl('foobar.xls'), 'application/vnd.ms-excel', 'xls', id='xls'),
-    ],
-)
+_url_formats = [
+    pytest.param(DocumentUrl('foobar.pdf'), 'application/pdf', 'pdf', id='pdf'),
+    pytest.param(DocumentUrl('foobar.txt'), 'text/plain', 'txt', id='txt'),
+    pytest.param(DocumentUrl('foobar.csv'), 'text/csv', 'csv', id='csv'),
+    pytest.param(
+        DocumentUrl('foobar.docx'),
+        'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+        'docx',
+        id='docx',
+    ),
+    pytest.param(
+        DocumentUrl('foobar.xlsx'),
+        'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+        'xlsx',
+        id='xlsx',
+    ),
+    pytest.param(DocumentUrl('foobar.html'), 'text/html', 'html', id='html'),
+    pytest.param(DocumentUrl('foobar.xls'), 'application/vnd.ms-excel', 'xls', id='xls'),
+]
+if sys.version_info[1] > 11:  # pragma: no branch
+    _url_formats.append(pytest.param(DocumentUrl('foobar.md'), 'text/markdown', 'md', id='md'))
```

Comment:
What is this for?

Reply:
Ah, my bad. I forgot to put a comment in there.